backend/README.md

```diff
@@ -29,19 +29,24 @@ registers and logins user
 returns:
 `{ "key": "some-string" }`
 
-### /pages/
+### /api/users/pages/
 
 GET
-`[{ "id": 1, "title": "", "author": "", "normal_url": "", "resolved_url": "", "date_saved": "", "date_published": "", "excerpt": "", "cover_image": "", "tags": "", "text": "" }]`
+`[{ "id": 1, "title": "", "author": "", "normal_url": "", "resolved_url": "", "date_saved": "", "date_published": "", "excerpt": "", "cover_image": "", "video": "", "audio": "", "tags": "", "text": "" }]`
 Returning a list of objects for each article that are in the database.
 
 POST
-`{ "id": 1, "title": "", "author": "", "normal_url": "", "resolved_url": "", "date_saved": "", "date_published": "", "excerpt": "", "cover_image": "", "tags": "", "text": "" },`
-Adding an article. `id` is auto generated. Only `title` is required.
+`{ "id": 1, "title": "", "author": "", "normal_url": "", "resolved_url": "", "date_saved": "", "date_published": "", "excerpt": "", "cover_image": "", "video": "", "audio": "", "tags": "", "text": "" }`
+Adding an article.
 
-### /pages/<int:id>/
+- `id` is auto generated.
+- `title` is required.
+- `date_saved` is default to today's date. No input necessary. Format is YYYY-MM-DD.
+
+### /api/users/pages/<int:id>/
 
 `<int:id>` is an integer id number auto incremented by the database.
+Actual url should look like this `/api/users/pages/1/`
```

Comment:
How would we specify which user's pages in particular?

Reply:
I will work on getting a user by id and it would eventually be something like users/1/pages/